test/index.js

```diff
@@ -105,10 +108,48 @@ describe('KeyringController', function () {
       await keyringController.persistAllKeyrings();
 
       const { vault } = keyringController.store.getState();
-      const keyrings = await mockEncryptor.decrypt(password, vault);
+      const keyrings = await mockEncryptor.decrypt(PASSWORD, vault);
       expect(keyrings.indexOf(unsupportedKeyring) > -1).toBe(true);
       expect(keyrings).toHaveLength(2);
     });
+
+    describe('on a manifest v3 build', function () {
```

Comment:
Nit: Maybe better to use the same terminology this project uses:

```suggestion
    describe(`when 'cacheEncryptionKey' is enabled`, function () {
```